plugins/x86/x86_prefix.ml

```diff
@@ -0,0 +1,53 @@
+open Core_kernel.Std
+open Or_error
+open Bap.Std
+
+type t = [
+  | `_F0 (** LOCK                *)
```

Comment:
I'm not sure that the underscore is a good and nice looking prefix (I got it that you need to have a prefix, as some prefixes doesn't have a predefined semantics, or their semantics is overloaded, so you just want to have type, that represents a subset of integers). My suggestion is to use `x` as the prefix, so that `0x67` is represented as 
```
`x67
```
That looks quite to nice to my taste, what do you think?